### DISCUSSION TITLE
When will Atom be able to accept tokens?

### DISCUSSION THREAD

**Discussion by CalebL11** [community/community#49731] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I downloaded Atom, and it was an earlier version. But all the links in Atom go to a blog post about sunsetting atom. Is Atom no longer available?

**Comment by radar-blip** [comment]
Atom was discontinued December 15th, 2022. 

https://www.howtogeek.com/810186/atom-text-editor-will-officially-die-later-this-year/

**Reply by CalebL11** [reply]
Will I still be able to use it?

**Reply by CalebL11** [reply]
By the way, thank you

**Reply by radar-blip** [reply]
You could still use it, yes. But it might be better to find an alternative.

You can read up on this, here:
https://www.ghacks.net/2022/06/09/githubs-atom-text-editor-will-be-retired-in-december/
